LICENSE

```diff
@@ -1,9 +1,9 @@
 MIT License
 
-Copyright (c) oemof developer group
+Copyright (c) 2015-2021, oemof developer group
```

Comment:
We do not need to add years here. As they need maintenance, I would skip this information.